src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -111,12 +111,12 @@ public class ServerWideDebugInfoPackageHandler : AdminRequestHandler
                                     continue; //skip writing local data, we do it separately
 
                                 await WriteDebugInfoPackageForNodeAsync(
```

Comment:
Probably better to execute all of those in parallel , you are already writing to memory stream anyway.
After you got it all, then write it to the archive.